docs/user_guide/backup/index.md

```diff
@@ -6,224 +6,75 @@ sidebar: sidebars/documentation.html
 
 # Backup
 
-## Prepare Backup
-
-In the following sections we will be using `/backup` directory as our targed directory that will hold backup archives. Let's prepare it now.
+In the following sections we will be using `/tmp/backup` directory as our targed directory that will hold backup archives.
 
 All the following commands are executed under `root` system account.
```

Comment:
To help line 9 and 11 stand out since they are prereqs for this, something like the following may help:

```
In the following sections, these assumptions are being made with respect to making the backup:

 * `/tmp/backup` will be used as the target for backups
 * All commands are executed as `root`